exercises/list-ops/.meta/HINTS.md

```diff
@@ -0,0 +1,5 @@
+This exercise is meant to be implemented with as few built-in functions as possible. This means that a backing array is allowed, but no functions from the `Array.prototype` should be used.
+
+- You don't need any function from `Array.prototype`, and only need the language feature to build construct and desconstruct arrays via destructuring as well as the array literal `[]`.
+- Most implementations using `for` loops also use `.length` on the backing array, or the `array[indexer]`. You don't need either!
```

Comment:
I think we should not talk about `implementations` in exercise README. I get the idea but the wording can be rephrased.